db/seeds/0_settings.rb

```diff
@@ -4,5 +4,5 @@
 
 seeder = SettingSeeder.new
 
-seeder.seed_setting(Setting::CountryCode, 'general_country_source_whitelist', [])
-seeder.seed_setting(Setting::IpRange, 'general_ip_whitelist', [])
```

Comment:
ah now I see why there are 4 settings in production.
in the seeder there's still the old names.
thx for fixing this!